crates/ef-testing/src/storage/eoa.rs

```diff
@@ -47,36 +45,22 @@ pub fn get_eoa_class_hash(
 }
 
 /// Initializes the EOA contract.
-/// Writes the kakarot address and the evm address to a hashmap.
+/// Returns the storage tuples for eoa contract.
 pub fn initialize_eoa(
     kakarot_address: FieldElement,
     evm_address: FieldElement,
-    destination: &mut HashMap<StarknetStorageKey, StarkFelt>,
-) -> Result<(), RunnerError> {
-    write_is_initialized(destination)?;
-    write_kakarot_address(kakarot_address, destination)?;
-    write_evm_address(evm_address, destination)?;
-    Ok(())
+) -> Result<Vec<(StarknetStorageKey, StarkFelt)>, RunnerError> {
+    let eoa_storage = vec![
+        get_is_initialized()?,
+        get_kakarot_address(kakarot_address)?,
+        get_evm_address(evm_address)?,
+    ];
+    Ok(eoa_storage)
 }
 
-/// Writes the kakarot address to a hashmap.
-fn write_kakarot_address(
+/// Returns the kakarot address storage tuple.
+fn get_kakarot_address(
```

Comment:
same for the `get_` i'd prefer `_storage`

Reply:
got it will update